Connected System Plug-in (CSP) Examples/Hello World/src/main/java/com/mycorp/helloworld/templates/HelloWorldConnectedSystemTemplate.java

```diff
@@ -4,15 +4,18 @@
 import com.appian.connectedsystems.simplified.sdk.configuration.SimpleConfiguration;
 import com.appian.connectedsystems.templateframework.sdk.ExecutionContext;
 import com.appian.connectedsystems.templateframework.sdk.TemplateId;
+import java.util.*;
 
 @TemplateId(name="HelloWorldConnectedSystemTemplate")
 public class HelloWorldConnectedSystemTemplate extends SimpleConnectedSystemTemplate {
 
   public static final String CS_PROP_KEY = "csProp";
+  public String PASSWORD = "password";
 
   @Override
   protected SimpleConfiguration getConfiguration(
       SimpleConfiguration simpleConfiguration, ExecutionContext executionContext) {
+    int x = 10;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "x". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "x" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)